docs/src/pages/components/tabs/HoverTabs.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+
+export default function CenteredTabs() {
```

Comment:
It should match the file name header

```suggestion
export default function HoverTabs() {
```